pkg/storage/unified/resource/client.go

```diff
@@ -17,9 +19,13 @@ import (
 	"github.com/grafana/grafana/pkg/apimachinery/identity"
 	"github.com/grafana/grafana/pkg/services/auth"
 	"github.com/grafana/grafana/pkg/services/authn/grpcutils"
+	"github.com/grafana/grafana/pkg/setting"
```

Comment:
hymmm -- I don't think this can be here (but also surprised the `pkg/services/auth` can!). This package has its own [go.mod](https://github.com/grafana/grafana/blob/main/pkg/storage/unified/resource/go.mod) so that we can build different implementations... see https://github.com/grafana/exp-unified-storage/blob/main/go.mod#L9

we should really add lint checking to make sure this fails!